--- zopfli-zopfli-1.0.1/Makefile.org	2017-10-13 01:27:55.643309889 +0000
+++ zopfli-zopfli-1.0.1/Makefile	2017-10-13 01:27:59.098309776 +0000
@@ -40,3 +40,11 @@
 # Remove all libraries and binaries
 clean:
 	rm -f zopflipng zopfli $(ZOPFLILIB_OBJ) libzopfli*
+
+install:
+	mkdir -p $(DESTDIR)/usr/lib64
+	mkdir -p $(DESTDIR)/usr/include
+	cp libzopfli.so.1.0.2 $(DESTDIR)/usr/lib64
+	ln -s libzopfli.so.1.0.2  $(DESTDIR)/usr/lib64/libzopfli.so
+	ln -s libzopfli.so.1.0.2  $(DESTDIR)/usr/lib64/libzopfli.so.1
+	cp src/zopfli/zopfli.h $(DESTDIR)/usr/include
\ No newline at end of file
